docs/GATK_GERMLINE_README.md

```diff
@@ -1,9 +1,48 @@
 # Kids First DRC Single Sample Genotyping Workflow
 Kids First Data Resource Center Single Sample Genotyping Workflow. This workflow closely mirrors the [Kids First DRC Joint Genotyping Workflow](https://github.com/kids-first/kf-jointgenotyping-workflow/blob/master/workflow/kfdrc-jointgenotyping-refinement-workflow.cwl).
-While the Joint Genotyping Workflow is meant to be used with trios, this workflow is meant for processing single samples.
-The key difference in this pipeline is a change in filtering between when the final VCF is gathered by GATK GatherVcfCloud and when it is annotated by VEP bcftools (see [Kids First DRC Germline SNV Annotation Workflow docs](https://github.com/kids-first/kf-annotation-tools/blob/v1.1.0/docs/GERMLINE_SNV_ANNOT_README.md) ).
-Unlike the Joint Genotyping Workflow, a germline-oriented [GATK hard filtering process](https://gatk.broadinstitute.org/hc/en-us/articles/360035890471-Hard-filtering-germline-short-variants) is performed and CalculateGenotypePosteriors has been removed.
-While somatic samples can be run through this workflow, be wary that the filtering process is specifically tuned for germline data.
+
+While the Joint Genotyping Workflow is meant to be used with whole genome
+sequenced trios, this workflow is meant for processing single samples from any
+sequencing experiment. The key difference between the different approaches is
+the filtering process.
+
+While somatic samples can be run through this workflow, be wary that the
```

Comment:
A good distinction - if one wanted to run a somatic or tumor sample for some reason, we can override the filters, right